app/controllers/orders_controller.rb

```diff
@@ -22,28 +22,31 @@ def new
   end
 
   def create
-    cart = Cart.new(session[:cart])
+    pending_loans = PendingLoan.new(session[:pending_loan])
     total_cost = 0
-    cart.projects.each do |project, quantity|
-      total_cost += project.price * quantity.to_i
+    pending_loans.projects.each do |project, loan_amount|
+      total_cost += project.price * loan_amount
     end
     @order = Order.create(
-      user_id: session[:user_id],
-      total_cost: total_cost,
-      status: 'ordered'
-      )
-    cart.projects.each do |project, quantity|
-      Loan.create(
+        total_cost: total_cost,
+        user_id: session[:user_id],
+        status: "completed")
+    pending_loans.projects.each do |project, loan_amount|
```

Comment:
Unused block argument - `loan_amount`. If it's necessary, use `_` or `_loan_amount` as an argument name to indicate that it won't be used.